content/en/user-guide/web-application/custom-local-endpoint/index.md

```diff
@@ -17,3 +18,11 @@ To ensure that the Web user interface can connect with your running LocalStack i
 To navigate this, we recommend you create a local TCP proxy server. The proxy server could listen on `127.0.0.1:4566` and forward all requests to your target endpoint where the LocalStack instance is running. You could leave the configuration in the Web user interface to use the default value, `https://localhost.localstack.cloud:4566`. We recommend [simpleproxy](https://manpages.ubuntu.com/manpages/trusty/man1/simpleproxy.1.html) or [proxy.py](https://github.com/abhinavsingh/proxy.py) as a way to implement this.
 
 An alternate workaround would be to re-point `localhost.localstack.cloud` to the IP address of your target machine by adding an entry to `/etc/hosts`. This is especially useful if you're accessing the LocalStack Web UI on a macOS/Linux-based machine.
+
+## Troubleshooting
+
+After successfully deploying LocalStack and mapping it to the internet at `https://myhost:4566`, you should be able to access the LocalStack Web Application and other exotic features, such as Resource Browsers, on your main machine without any problems. However if you are unable to access the LocalStack Web Application, start by checking the **Networking tab** of your browser's developer console to gather more information about the connection issue.
+
+It's possible that your browser is preventing the connection between HTTPS pages and non-encrypted HTTP endpoints. To overcome this, try configuring `https://192.168.9.88` as the target endpoint in the profile settings on `app.localstack.cloud`. Keep in mind that doing so may result in SSL certificate validation problems. To handle this, open the provided URL in your browser and manually accept the certificate.
```

Comment:
whats `https://192.168.9.88` ? shouldn't `https://localhost.localstack.cloud:4566` suffice, which is the default nowadays?